README.md

```diff
@@ -1,138 +1,64 @@
-
-## Development
-
-Development environment is set up to enforce good practices in JS development (static code analysis, style formatting, code coverage check). 
-
-Some general guidelines for different type of assets are listed below. 
-
-### Static files
-
-- add new assets to dedicated folder for specific type 
-- when removing assets verify that it is not used anywhere
-- if relevant and possible add documentation on asset usage (eg. in README.md file in the same folder as asset)
-
-### JS API
-
-- add new code to `src` folder
-- remember to export new files in `src/index.js` (otherwise the new code will not be available in the distribution package)
-- build: `npm run build` (production build) or `npm run dev` (automatic rebuilding)
-- test: `npm test` (static analysis, code style check, documentation check and unit testing with [Jest](https://jestjs.io/en/) testing framework) 
-- use [prettier](https://prettier.io/) and [eslint](https://eslint.org/) during development
-- document your code (we are using [JSdoc block tags](https://jsdoc.app/#block-tags) as documentation is auto-generated using [documentation.js](http://documentation.js.org))
+To run any NPM script for specific package from root package, run: 
+```
+npm run <script-name> --workspace packages/<package-name>
+```
 
+To run any NPM script for all packages only if the script exists, run:
+```
+npm run <script-name> --workspaces --if-present
+```
 
 ## Debugging
 
-### Internal
-
-There is no standalone application for `cloudify-ui-common`, so the best way to debug some portion of it internally is using Jest unit tests (see: [test](./test) folder).
-
-### External
-
-If you want to develop/debug `cloudify-ui-common` from the package user side (eg. from [cloudify-stage](https://github.com/cloudify-cosmo/cloudify-stage)), then instead of using `cloudify-ui-common` package from NPM registry, you can:
- 1. Use `npm link cloudify-ui-common <local-path-to-cloudify-ui-common>` command in your package user project,
- 1. Build `cloudify-ui-common` project (see: [Development](#development) section), 
- 1. Run package user project and see changes applied locally in `cloudify-ui-common`.   
+If you want to develop/debug `cloudify-ui-common-<package-name>` from the package user side (eg. from [cloudify-stage]
+(https://github.com/cloudify-cosmo/cloudify-stage)), then instead of using `cloudify-ui-common-<package-name>` package from NPM 
+registry, you can:
+1. Use `npm link cloudify-ui-common-<package-name> <local-path-to-cloudify-ui-common-<package-name>>` command in your package user project,
+1. Build `cloudify-ui-common-<package-name>` package (see: [Development](#development) section),
+1. Run package user project and see changes applied locally in `cloudify-ui-common-<package-name>` package.
 
-
-## Deployment
-
-Cloudify UI Common library is published in [NPM](https://www.npmjs.com) registry. See [cloudify-ui-common@npm](https://www.npmjs.com/package/cloudify-ui-common).
+## Publishing
```

Comment:
In this section more changes are probable once I get to publishing mechanism update.